src/CommandingOfficers/AW3/AW3ValueFinders.java

```diff
@@ -0,0 +1,73 @@
+package CommandingOfficers.AW3;
+
+import CommandingOfficers.Commander;
+import CommandingOfficers.IValueFinder;
+import Engine.UnitActionFactory;
+import Units.Unit;
+import Units.UnitModel;
+
+public abstract class AW3ValueFinders
+{
+  public static class CostValueFinder implements IValueFinder
+  {
+    int maxDamage = 3;
+    @Override
+    public int getValue(Commander attacker, Unit unit)
+    {
+      // Ignore Oozium, if we ever add those
```

Comment:
Why? Also, should Sturm's version have a similar note?

Reply:
Because the carts do that, and Oozium is immune to mass damage of all flavors.
Sturm never coexisted with Oozium, so eh.

Reply:
Sturm can mass damage Oozium, got it. Sturm OP.